lib/api/bucketGetTagging.js

```diff
@@ -0,0 +1,105 @@
+const { metadataValidateBucket } = require('../metadata/metadataUtils');
+const util = require('node:util');
+const collectCorsHeaders = require('../utilities/collectCorsHeaders');
+const { checkExpectedBucketOwner } = require('./apiUtils/authorization/bucketOwner');
+const { pushMetric } = require('../utapi/utilities');
+const { errors, s3middleware } = require('arsenal');
+const escapeForXml = s3middleware.escapeForXml;
+
+//  Sample XML response:
+/*
+    <Tagging>
+        <TagSet>
+            <Tag>
+                <Key>string</Key>
+                <Value>string</Value>
+            </Tag>
+            <Tag>
+                <Key>string</Key>
+                <Value>string</Value>
+            </Tag>
+        </TagSet>
+    </Tagging>
+*/
+
+/**
+ * Convert Versioning Configuration object of a bucket into xml format.
+ * @param {array} tags - set of bucket tag keys and values
+ * @return {string} - the converted xml string of the versioning configuration
+ */
+function convertToXml(tags) {
```

Comment:
Now that I think about it, this function name can be better, something like `tagsToXml`, as the name itself doesn't reference what its converting, so it could be anything.